src/components/createTarget/index.js

```diff
@@ -14,14 +14,16 @@ import { useCreateTargetMutation, useGetTargetsQuery } from 'services/target/tar
 
 import sideBarIcon from 'assets/aim-mark.png';
 
-const NewTarget = () => {
+const NewTarget = ({ ...sendLatLng }) => {
   const t = useTranslation();
+  let position = [sendLatLng.sendLatLng.lat, sendLatLng.sendLatLng.lng];
```

Comment:
Does this still work? You're spreading the `sendLatLng` object in the props, should you access `lat` or `lng` directly? Or maybe `sendLatLng.lat` instead of calling two times `sendLatLng`? I'm not sure, I'm just asking 😄